hpx/util/invoke.hpp

```diff
@@ -127,6 +127,23 @@ namespace hpx { namespace util
         };
     }
 
+    /// Invokes the given functional type f with the content of
+    /// the argument pack vs
+    ///
+    /// \param f A functional type providing an `operator()`
+    ///          or function pointer. If f is a member function pointer,
+    ///          the first argument in the pack will be treated as
+    ///          the callee (this object).
+    ///
+    /// \param vs An arbitrary pack of arguments
+    ///
+    /// \returns The result of the function when it's called with
+    ///          the given argument types.
+    ///
+    /// \throws std::exception thrown from a function
+    ///         invocation of f with the argument types vs.
```

Comment:
Doxygen provides it as a part of its API: [`\throws <exception-object> { exception description }`](https://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdthrows).